spec/yaml/swiftnav/sbp/integrity.yaml

```diff
@@ -0,0 +1,94 @@
+# Copyright (C) 2015-2021 Swift Navigation Inc.
+# Contact: https://support.swiftnav.com
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+package: swiftnav.sbp.integrity
+description: Over-the-air Integrity Messages
+stable: True
+public: False
+include:
+  - types.yaml
+  - gnss.yaml
+definitions:
+
+ - OrbitClockBound:
```

Comment:
In the excel file Orbit Bounds and Clock Bounds are in separate groups. But I think it make more sense to have them combined in one structure, since the satellite ID is only present in the first group.